neurom/io/hdf5.py

```diff
@@ -146,6 +146,40 @@ def find_parent_id(point_id):
                           find_parent_id(i))
                          for i, p in enumerate(points)])
 
+    @staticmethod
+    def remove_duplicate_points(points, groups):
+        ''' Removes the duplicate points
+            from the beginning of a section,
+            if they are present in points-groups representation.
+            Returns points, groups with unique points.
```

Comment:
Could you make the docstring compatible with the google style so it renders well?